llpc/test/shaderdb/core/TestForceNonUniformResourceIndex.frag

```diff
@@ -0,0 +1,30 @@
+#version 450
+
+#extension GL_EXT_nonuniform_qualifier : require
+
+layout(set = 0, binding = 0) buffer Data
+{
+    vec4 color;
+} data[];
+
+layout(location = 0) out vec4 FragColor;
+layout(location = 0) in flat int index;
+void main()
+{
+  FragColor = data[index].color;
+}
+
+// BEGIN_NOTFORCENURITEST
+// RUN: amdllpc -v %gfxip %s --force-non-uniform-resource-index-stage-mask=0x00000000 | FileCheck -check-prefix=NOTFORCENURITEST %s
```

Comment:
I think we started to put the test lines at the top for all new tests.
Also, I’m not sure what the intention behind the `BEGIN_*` and `END_*` lines are (@amdrexu, do you know?), but usually they are `BEGIN_SHADERTEST` and `END_SHADERTEST`.

Reply:
Yes, we usually use BENGIN_SHADERTEST and END_SHADERTEST. No other delimiters.